build.gradle

```diff
@@ -1,206 +1,207 @@
-                   ':persistence:model',
-                   ':shared:lib-camel-connector',
-                   ':service:provider',
-                   ':service:spi',
-                   ':svc-lighthouse-api',
-                   ':console']
-    dockerModules = [':app',
-                     ':db-init',
-                     ':service-python:assessclaimdc7101',
-                     ':service-python:assessclaimdc7101v2',
-                     ':service-python:assessclaimdc6602',
-                     ':service-python:pdfgenerator',
-                     ':svc-lighthouse-api',
-                     ':console']
+  // TODO: set back to true; temporarily disable test code coverage enforcement
+  jacoco_enforce_violations = false
+  jacoco_minimum_coverage = 0.8
+  codeModules = [
+    ':app',
+    ':api',
+    ':controller',
+    ':persistence:model',
+    ':shared:lib-camel-connector',
+    ':service:provider',
+    ':service:spi',
+    ':svc-lighthouse-api',
+    ':console'
+  ]
+  dockerModules = [
+    ':app',
+    ':db-init',
+    ':service-python:assessclaimdc7101',
+    ':service-python:assessclaimdc7101v2',
+    ':service-python:assessclaimdc6602',
+    ':service-python:pdfgenerator',
+    ':svc-lighthouse-api',
+    ':console'
+  ]
 }
 
 // Used by `release` task to create a new version
 scmVersion {
-    // never connect to remote
-    localOnly = true
-    useHighestVersion = true
-
-    hooks {
-        pre("fileUpdate", [file: "app/src/main/java/gov/va/vro/config/propmodel/Info.java",
-                           pattern: {v,p -> /version = "v.*"/},
-                           replacement: {v, p -> "version = \"v$v\""}])
-        pre("fileUpdate", [file: "helmchart/Chart.yaml",
-                           pattern: {v,p -> /appVersion: .*/},
-                           replacement: {v, p -> "appVersion: $v"}])
-        pre("commit")
-    }
+  // never connect to remote
+  localOnly = true
+  useHighestVersion = true
+
+  hooks {
+    pre("fileUpdate", [file       : "app/src/main/java/gov/va/vro/config/propmodel/Info.java",
+      pattern    : { v, p -> /version = "v.*"/ },
+      replacement: { v, p -> "version = \"v$v\"" }])
+    pre("fileUpdate", [file       : "helmchart/Chart.yaml",
+      pattern    : { v, p -> /appVersion: .*/ },
+      replacement: { v, p -> "appVersion: $v" }])
+    pre("commit")
+  }
 }
 
 allprojects {
-    // https://axion-release-plugin.readthedocs.io/en/latest/configuration/basic_usage
-    project.version = scmVersion.version
-}
-
-// Support pre-commit hooks, need spotlessCheck and spotlessApply
-def spotlessCheck = tasks.register('spotlessCheck') {
-    def tb = taskBuilder('spotlessCheck')
-    codeModules.each { dependsOn tb(it) }
-}
-
-def spotlessApply = tasks.register('spotlessApply') {
-    def tb = taskBuilder('spotlessApply')
-    codeModules.each { dependsOn tb(it) }
+  // https://axion-release-plugin.readthedocs.io/en/latest/configuration/basic_usage
+  project.version = scmVersion.version
 }
```

Comment:
Adding the `spotless` plugin adds these 2 tasks.
(Note: we don't (currently) use pre-commit hooks, so removing these lines has no impact)